Mathlib/Order/Filter/Bases.lean

```diff
@@ -826,6 +826,11 @@ lemma eventually_prod_self_iff {r : α → α → Prop} :
     (∀ᶠ x in la ×ˢ la, r x.1 x.2) ↔ ∃ t ∈ la, ∀ x ∈ t, ∀ y ∈ t, r x y :=
   mem_prod_self_iff.trans <| by simp only [prod_subset_iff, mem_setOf_eq]
 
+/-- A version of `eventually_prod_self_iff` that is more suitable for forward rewriting. -/
+lemma eventually_prod_self_iff' {r : α × α → Prop} :
```

Comment:
Note that this lemma is no longer used in this PR.

Reply:
I think it's still a nice one to have, and in general we have these two variants for theorems about quantifiers.